packages/camera/camera/ios/Classes/FLTThreadSafeFlutterResult.h

```diff
@@ -0,0 +1,46 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <Flutter/Flutter.h>
+
+/**
+ Wrapper object that always delivers the result on the main thread.
+ */
+@interface FLTThreadSafeFlutterResult : NSObject
+
+/**
+ Initialize with a FlutterResult object.
+ @param result The FlutterResult object that the result will be given to.
+ */
+- (id _Nonnull)initWithResult:(FlutterResult _Nonnull)result;
+
+/**
+ Send a successful result without any data.
+ */
+- (void)success;
+
+/**
+ Send a successful result with data.
+ @param data Result data that is send to the Flutter Dart side.
+ */
+- (void)successWithData:(id _Nonnull)data;
+
+/**
+ Send an error as result
```

Comment:
s/error/NSError/

Reply:
What do you mean?
"Sends an NSError as result" Or "Sends s/error/NSError/"?